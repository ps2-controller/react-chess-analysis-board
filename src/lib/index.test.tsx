import React from 'react'
import renderer from 'react-test-renderer'
import { describe, expect, test } from 'vitest'
import PGNBoard from './index'

describe('PGNBoard', () => {
  test('PGNBoard component renders correctly', () => {
    const component = renderer.create(
      <PGNBoard />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })

  test('The greetee prop works', () => {
    const component = renderer.create(
      <PGNBoard />
    )

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})