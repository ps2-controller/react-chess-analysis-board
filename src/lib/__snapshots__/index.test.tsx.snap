// Vitest Snapshot v1

exports[`HelloWorld > HelloWorld component renders correctly 1`] = `
<div>
  Hello, 
  World
  !
</div>
`;

exports[`HelloWorld > The greetee prop works 1`] = `
<div>
  Hello, 
  Universe
  !
</div>
`;

exports[`PGNBoard > PGNBoard component renders correctly 1`] = `null`;

exports[`PGNBoard > The greetee prop works 1`] = `null`;
